soroban-sdk/src/testutils.rs

```diff
@@ -67,3 +70,21 @@ pub trait Accounts {
     /// Remove an account.
     fn remove(&self, id: &AccountId);
 }
+
+pub fn random_id() -> [u8; 32] {
```

Comment:
What type of `id` is this intended to be? Or is this intended to be just a random 32-byte byte slice? It feels a bit arbitrary.

Instead of making this about "ids" how about we change it to:
```suggestion
pub fn random_id<const N: usize>() -> [u8; N] {
```

Reply:
Updated.